### PR TITLE
Release v0.7.0

### PR DESCRIPTION
**Pull request by khoroshevskyi**
What was changed here?
- [x] add changelog
- [x] update version
- [x] update pypi new release config
- [x] add message here:)

Changes:
- Added `pop` to project table and annotation model [#107](https://github.com/pepkit/pepdbagent/issues/107)
- Added `forked_from` feature [#73](https://github.com/pepkit/pepdbagent/issues/73)
- Switched to pydantic2 [#105](https://github.com/pepkit/pepdbagent/issues/105)
- Updated requirements (psycopg2 -> psycopg3) [#102](https://github.com/pepkit/pepdbagent/issues/102)
- Added sample module that contains functionality for updating, adding, and deleting samples in the project separately [#111](https://github.com/pepkit/pepdbagent/issues/111)
- Added user and favorite tables with functionality [#104](https://github.com/pepkit/pepdbagent/issues/104)
- Updated the sample update method when updating the whole project. Following this change, samples are updated without changing the ID in the database

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ agent.project.create(prj_obj, namespace, name, tag)
...
 update_dict1 = {"is_private" = True}
 update_dict2 = {"is_private" = True, name = "new_name"}
-update_dict3 = {"project" = prj_obj, "is_private"=True}
+update_dict3 = {"project" = prj_obj, "is_private" = True}
 update_dict4 = {"project" = prj_obj}
 update_dict4 = {"tag" = "new_tag"}
 
 # after creation of the dict, update record by providing update_dict and namespace, name and tag:
-agent.project.update(update_dict, namespace, name, tag)
+agent.project.update(update_dict, namespace, name, tag, )
```
Comment (nleroy917) [inline docs/README.md:53]: Trailing comma?

File: pepdbagent/modules/view.py [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+                Projects.namespace == view_dict.project_namespace,
+                Projects.name == view_dict.project_name,
+                Projects.tag == view_dict.project_tag,
+            )
+        )
+
+        with Session(self._sa_engine) as sa_session:
+            project = sa_session.scalar(project_statement)
+            if not project:
+                raise ProjectNotFoundError(
```
Comment (nleroy917) [inline pepdbagent/modules/view.py:166]: Why are we looking for existing projects if this is the `create()` function?

**Review by nleroy917 — COMMENTED** [review]
Its a lot of code, but at a cursory glance, it seems ok

File: pepdbagent/modules/view.py (hunk 2 above)
Comment (khoroshevskyi) [reply pepdbagent/modules/view.py:166]: We are creating view, not project

**Pull request merged**